RELEASE_NOTES.md

```diff
@@ -63,7 +63,7 @@ See the samples coming with the distribution.
 
 To see the list of available flags, run `kotlinc -h`.
 
-For documentation on C interoperability stubs see INTEROP.md.
+For documentation on C interoperability stubs see [INTEROP.md](INTEROP.md).
```

Comment:
Please use full URL, like below.

Reply:
Oh yeah, thanks.
It's part of the binaries. Should be the full url, you are right 👍